stock_account_product_run_fifo_hook/hooks.py

```diff
@@ -62,11 +62,12 @@ def _run_fifo_new(self, quantity, company):
 
         # Update the standard price with the price of the last used candidate,
         # if any.
-        if new_standard_price and self.cost_method == "fifo":
+        # START HOOK update standard price
+        if self._price_updateable(new_standard_price):
```

Comment:
Maybe you should add also something to check this function exists like here: https://github.com/ForgeFlow/stock-logistics-workflow/pull/3/files#diff-eb0783cb5bfd7be2476efed449d4f7284911d91d693449d0410ed93b24206c61R11

Reply:
Sure, can you show me an example of how to do that? I don't remember to do anything similar in the past Thanks in advance

Reply:
I think the example is in the same file xD sorry

Reply:
Does it look correct now?